pages/reflections.md

```diff
@@ -1,11 +1,47 @@
+
+* _Delving deeper into historical and political literature surrounding critical bills and events affecting immigration:_
+
+   While each of us as students of history have the skills to take deep dives into historical literature, we certainly did not have the time to do so. The number of topics encompassed within this project had enough material to write multiple dissertations on. As such, our historical writing on the project had to be kept to the headline bullet points in a somewhat simplistic cause and effect way. This was sufficient for our purposes but, as with data compilation, more detailed historical context for the changes in immigration would be required to gain more insight into this topic.
+
+* _Lastly, more diverse modes of visualization:_ 
 
-You should talk about where you group encountered the most difficulties. What was the hardest about scraping, or visualizing, or locating historiography? Did you have problems communicating or sharing files on GitHub?
+   Our project utilized area charts to display the changing immigration data: a perfect medium for conveying the type of data we had. Although I believe the area chart to be optimal for conveying immigration data, it is limited in its accessibility and visceral understanding. Additional graphing types, such as a map over time, could provide a far more intuitive medium for the casual observer. Having a map as part of our graph suite was our initial intention, however we were unable to do so due to a limitation with the data formatting making it impossibly tedious. Had we had more time, and better compiled data, we would be able to create more diverse visualizations to increase the overall polish and accessibility of the project.
 
-What problems stopped you dead in the water? Did you have to change course mid-project to account for data that you could not get, or was not what you thought?
+---
 
-Moving on from problems, let's the reader on a positive note. What worked best, where did you think the strength of the project was? What were the most useful historical insights? What were the most useful tools?
+   I think the strength of our project lies in our bread and butter area chart. Our visualizations clearly and accurately showcase the macro situation with immigration and can give a solid idea about the history of US immigration at a glance. I had a vision of those graphs when we first undertook this project, and they turned out just as I had anticipated them to. Frankly, I could not be happier with how they turned out, and I think that we built upon that graph nicely by overlaying world events and legislation that impacted immigration to the US.
+				
+Lines of inquiry that came up for us that we lacked the ability to resolve were legion, for they were many:
```

Comment:
Good job on this page, I think that you highlighted a good range of not only limitations but also possibilities.